package.json

```diff
@@ -102,24 +102,35 @@
     ],
     "rules": {
       "angular/angularelement": "error",
-      "angular/di": "error",
-      "angular/di-order": "error",
+      "angular/di": "warn",
+      "angular/di-order": "warn",
```

Comment:
This is mostly stylistic, so it can stand to be a warning for now.  We intend to bite the bullet at some point and flip this back to error.